src/graphql/mintNFT.ts

```diff
@@ -147,7 +156,21 @@ export const MintNft = mutationField('mintNft', {
 
     // Upload NFT Metadata, use metadata from api call, assume NFT Standard format
     try {
-      uri = await metaplex.nfts().uploadMetadata(args.nftMetadata);
+      if (args.nftMetadata) {
```

Comment:
We support either file or nft metadata right? 
You adding this here first to test? I think it looks fine.

Reply:
right, either file json, or metadata as args